code/headers/dummy_controller.hpp

```diff
@@ -8,9 +8,12 @@ namespace r2d2::manual_control {
         // initializing all buttons, sliders and joysticks at their base (0)
         bool buttons[12] = {};
         unsigned char sliders[2] = {};
-        joystick_value_s joysticks[2] = {};
+        joystick_value_c joysticks[2] = {};
```

Comment:
Personally, I would like to see std::array instead of c style arrays same goes for the bool array and for the char array however I see the char array as an exception.

Reply:
I agree, changed